api/src/Entity/Activity.php

```diff
@@ -241,6 +214,20 @@ public function removeScheduleEntry(ScheduleEntry $scheduleEntry): self {
         return $this;
     }
 
+    /**
+     * ActivityResponsibles.
+     */
+
+    /**
```

Comment:
```suggestion
    /**
```

Reply:
Yeah, can remove. Was an attempt to start improving the overview of the Entity methods, alas, not a very successful one. 😄